src/services/activityService.ts

```diff
@@ -0,0 +1,105 @@
+import { ActivityOptions, ActivityType, Client } from "discord.js";
+import { selectRandom } from "../utils";
+
+type ActivityChoices = {
+    activity: ActivityType;
+    options: Array<string>;
+};
+
+const CHOICES: Array<ActivityChoices> = [
+    {
+        activity: "WATCHING",
+        options: [
+            "you... FROM INSIDE THE HOUSE",
+            "Jan Quadrant Vincent 16",
+            "Adventures of Baby Legs and Regular Legs",
+            "Avatar 86: This Time The Trees Are Real Mad",
+            "Seinfeld with Abyssal dub"
+        ]
+    },
+    {
+        activity: "PLAYING",
+        options: [
+            "Starcraft 6",
+            "Starfox 65",
+            "Phasmophob - OH SHIT HIDE",
+            "Bird Law Simulator 2037",
+            "Call Of Dookie",
+            "Sturgeon Simulator",
+            "Untitled Spruce Game",
+            "Pumpkin Carving Simulator 2020"
+        ]
+    },
+    {
+        activity: "LISTENING",
+        options: [
+            "Nefertiti's Fjord",
+            "We Didn't Start The Fire, but at 50% speed",
+            "John Malkovich gurgling marbles",
+            "Spooky Scary Skeletons"
+        ]
+    },
+    {
+        activity: "STREAMING",
+        options: ["a literal stream", "ice cubes melting in the sun", "my neighbor napping in a hammock"]
+    }
+];
+
+export class ActivityService {
```

Comment:
So, this actually works with no extra effort in the tsyringe? Like, I would think this would need the `@singleton` tag or something.

Reply:
oh yeah I forgot the @singleton annotation. It's likely just creating a new instance every time it's resolved, so I'll make sure it doesn't do that